casper/src/main/scala/coop/rchain/casper/util/comm/DeployService.scala

```diff
@@ -58,8 +58,8 @@ class GrpcDeployService(host: String, port: Int, maxMessageSize: Int)
       else Left(new RuntimeException(response.status))
     }
 
-  def visualizeBlocks(q: BlocksQuery): Task[Either[Throwable, String]] =
-    stub.visualizeBlocks(q).map { response =>
+  def visualizeDag(q: VisualizeDagQuery): Task[Either[Throwable, String]] =
```

Comment:
How easily can I DOS the node by calling `visualizeDag(depth = -1)` repeatedly? :smiling_imp: